### PR TITLE
Guest list

### PR DESCRIPTION
**Pull request by Natis23**

### CODE REVIEW

File: client/screens/HomeScreen.js [hunk 1]
```diff
@@ -1,58 +1,86 @@
...
+            </Text>
+          </TouchableOpacity>
+    {/*       <Text>Email: {auth.currentUser?.email}</Text> */}
+          <TouchableOpacity onPress={() => navigation.navigate("Guest")}>
+            <Text> 
+              View All Guests
+            </Text>
+          </TouchableOpacity>
+
+<<<<<<< HEAD
```
Comment (E-N-House) [inline client/screens/HomeScreen.js:32]: shoule this be here

File: client/screens/HomeScreen.js [hunk 2]
```diff
@@ -1,58 +1,86 @@
...
+      </TouchableOpacity>
+      <Text style={{ fontSize: 12, textAlign: "center", marginTop: 10 }}>
+        <Text
+          style={{ color: "darkblue", fontWeight: "bold" }}
+          onPress={() => navigation.navigate("Users")}
+        >
+          View All Users
+        </Text>
+      </Text>
+>>>>>>> 8342f16 (Single page Working and guest profile)
```
Comment (E-N-House) [inline client/screens/HomeScreen.js:60]: shouldn't be there?

File: client/screens/HomeScreen.js [hunk 3]
```diff
@@ -1,58 +1,86 @@
...
             <Text style={{ color: 'darkblue', fontWeight: 'bold' }} onPress={() => navigation.navigate("Users")}>
                 View All Users
             </Text>
         </Text>
+        <Text style={{ fontSize: 12, textAlign: 'center', marginTop: 10 }}>
+            <Text style={{ color: 'darkblue', fontWeight: 'bold' }} onPress={() => navigation.navigate("Create Event")}>
+                Create New Event
+            </Text>
+        </Text>
+=======
```
Comment (E-N-House) [inline client/screens/HomeScreen.js:47]: shouldn't be there

File: client/screens/LoginScreen.js [hunk 4]
```diff
@@ -1,8 +1,8 @@
 import { KeyboardAvoidingView, StyleSheet, Text, TextInput, TouchableOpacity, View } from 'react-native'
 import React, { useEffect, useState } from 'react'
-import { auth } from '../../firebase'
+import { auth } from '../firebase'
```
Comment (E-N-House) [inline client/screens/LoginScreen.js:3]: adjust

File: screens/HomeScreen.js
Comment (E-N-House) [inline screens/HomeScreen.js:1]: Wad this supposed to be deleted

**Comment by Natis23** [comment]
Updated

**Review by E-N-House — APPROVED** [review]

**Pull request merged**